.circleci/config.yml

```diff
@@ -96,6 +96,7 @@ jobs:
       - run: *upgrade-git
       - run: *setup
       - run: *android-cordova
+      - run: *install-sfdx
```

Comment:
forcehybrid-ios and forcehybrid-android jobs are failing on CI because the hybridlwc app can't be built.